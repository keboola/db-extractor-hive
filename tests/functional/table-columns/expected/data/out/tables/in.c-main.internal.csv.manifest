{
    "destination": "in.c-main.internal",
    "incremental": false,
    "metadata": [
        {
            "key": "KBC.name",
            "value": "internal"
        },
        {
            "key": "KBC.sanitizedName",
            "value": "internal"
        },
        {
            "key": "KBC.schema",
            "value": "default"
        },
        {
            "key": "KBC.catalog",
            "value": "HIVE"
        },
        {
            "key": "KBC.type",
            "value": "table"
        }
    ],
    "columns": [
        "price",
        "product_name"
    ],
    "column_metadata": {
        "price": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "FLOAT"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "price"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "price"
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 2
            }
        ],
        "product_name": [
            {
                "key": "KBC.datatype.nullable",
                "value": true
            },
            {
                "key": "KBC.datatype.basetype",
                "value": "STRING"
            },
            {
                "key": "KBC.datatype.default",
                "value": ""
            },
            {
                "key": "KBC.sourceName",
                "value": "product_name"
            },
            {
                "key": "KBC.sanitizedName",
                "value": "product_name"
            },
            {
                "key": "KBC.uniqueKey",
                "value": false
            },
            {
                "key": "KBC.ordinalPosition",
                "value": 1
            }
        ]
    }
}