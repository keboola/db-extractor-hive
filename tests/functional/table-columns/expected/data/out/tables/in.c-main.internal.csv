"12.34","car1"
"56.78","car2"
